server/utils/customer.js

```diff
@@ -15,7 +15,7 @@ exports.getUsers = function(req, res){
 }
 
 exports.getCompanies = function(req, res){
-  Company.find({}).then(companies=>{
+  Company.find({}).then((err,companies)=>{
```

Comment:
@shivanggarg1998 no need to catch the error here, the errors are handled by promises themselves.